### ISSUE TITLE
custom dataset

### ISSUE THREAD

**Issue by lzzlxxlsz** [samet-akcay/ganomaly#64]
Hello，I set the same structure of the data ,and the settings:     
   
        self.parser.add_argument('--dataset', default='dfh', help='folder | cifar10 | mnist ')
        self.parser.add_argument('--dataroot', default='.data/dfh/', help='path to dataset')
my dataset name is dfh,and the file of dfh contains two folders : train and test.the train file contains one file of 0 , 0  file include the normal pictures .

However,when I run the train.py ,it turns the error:
FileNotFoundError: [WinError 3] 系统找不到指定的路径。: '.data/dfh/train'

thanks

**Comment by Coderxiaobai1** [comment]
you do not need to set the default option --dataroot to '.data/dfh' , just let it be ' '.

**Comment by lzzlxxlsz** [comment]
thanks a lot ,another question turns.    
assert isize % 16 == 0, "isize has to be a multiple of 16"
AssertionError: isize has to be a multiple of 16


my dataset size is 200x200,and I set the :
        self.parser.add_argument('--isize', type=int, default=200, help='input image size.')
Must this isize number be a multiple of 16 ?

**Comment by Coderxiaobai1** [comment]
of course, it depends the generate's input setting. if not , it will never come out of the while loop

**Comment by Coderxiaobai1** [comment]
In fact,  i think it must be a series like [4,8,16,32,64,128,256,512...]

**Comment by lzzlxxlsz** [comment]
thanks ,another question.If my dataset size is not the multiple of 16,before I train the modle ,should I adjust the dataset size to be the multiple of 16?

> In fact, i think it must be a series like [4,8,16,32,64,128,256,512...]
你好，你在自己的数据集上跑通 了吗

**Comment by Coderxiaobai1** [comment]
there is no need to do that, you can just set --isize to 128 as long as it is smaller than 200.(your dataset)

**Comment by lzzlxxlsz** [comment]
thanks，how about the result for your dataset?

**Comment by Coderxiaobai1** [comment]
i haven't done much since i'm still exploring it.

**Comment by lzzlxxlsz** [comment]
OK,thanks a lot

**Comment by Coderxiaobai1** [comment]
please let me know if you can run the demo well on your dataset cause i am haunted much these days.😭

**Comment by lzzlxxlsz** [comment]
I run the train.py but there is nothing display.only the output folder has some loss_log.txt. does it mean  runs succussfully?

**Comment by Coderxiaobai1** [comment]
you can start the visdom server to have a look when in the training process

**Comment by Viki-researcher** [comment]
> you do not need to set the default option --dataroot to '.data/dfh' , just let it be ' '.
hi，why in my train the AUC is always nan,but the train generate picture is not too bad

**Comment by Linranran** [comment]
@17695996717  because there are only OK or NG samples in the test.

**Comment by Linranran** [comment]
@17695996717  the test should have the NG and OK samples simultaneously
